vars/cypress.groovy

```diff
@@ -22,6 +22,5 @@ def call(String app = null, String tag = null) {
 def cleanUp() {
     sh """
```

Comment:
Can't we just do `sh "docker-compose down || :"` now, or something similar (my groovy syntax-fu is lacking)